xtext-website/documentation/103_domainmodelnextsteps.md

```diff
@@ -339,7 +339,39 @@ This tutorial is about testing the parser, the linker and the validator for the
     }
     ```
 
-3.  After saving the Xtend file, it is time to run the test. Select *Run As &rarr; JUnit Test* from the editor's context menu.
+3. The [CompilationTestHelper]({{site.src.xtext_extras}}/org.eclipse.xtext.xbase.testing/src/org/eclipse/xtext/xbase/testing/CompilationTestHelper.java) utility class comes in handy while unit testing the custom generator implementations:
```

Comment:
does hold true for java targets only

Reply:
@cdietrich I have already successfully used the `CompilationTestHelper` class to test generators that has nothing to do with Xbase/Java. e.g I implemented a Graphviz `*.dot` file generator for the Xtext Simple Arithmetics example, and used the `CompilationTestHelper` class while implementing [ArithmeticsDotGeneratorTest](https://github.com/itemis/itemis-blog/blob/gef_graphical_views_on_xtext_dsls/SimpleArithmeticsExample/org.eclipse.xtext.example.arithmetics.tests/src/org/eclipse/xtext/example/arithmetics/tests/ArithmeticsDotGeneratorTest.xtend) test cases.

For that reason I would leave the documentation as general as possible and would not restrict if for testing java targets. Do you agree?

Reply:
ok

Reply:
Thank you! Could someone approve this contribution?